client/src/components/Connections/Connections.js

```diff
@@ -0,0 +1,77 @@
+
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom'
+import "./Connections.css";
+import ProfilePicture from '../ProfilePicture/ProfilePicture'
+
+
+// Class Based React Component
+class Connections extends Component {
+  constructor(props) {
+    super(props);
+
+    // Default CSS class to apply to the Component
+    this.state = {
+      Heading: "Connections"
+    };
+  }
+  // Runs after Component is loaded in the broswer
+  componentDidMount() {
+
+    if (this.props.userViewing === true) {
+      this.setState({ Heading: "My Connections" })
+    } else if (this.props.userViewing === false) {
+      this.setState({ Heading: "Her Connections" })
+    } else if (!this.props.userViewing) {
+      this.setState({ Heading: "Members" })
```

Comment:
Members is not rendering in your component as a header. The Member component still says "Her Connections". See if you can get this working.